filterUsers.js

```diff
@@ -0,0 +1,124 @@
+let users = [
+    {
+        userName: "Mario Rossi",
+        competenceUser: [
+            {
+                nameComp: 'java',
+                area: 'linguaggi programmazione',
+                level: 3
+            },
+            {
+                nameComp: 'Linux',
+                area: 'Sistemi operativi',
+                level: 2
+            }
+        ]
+    },
+    {
+        userName: "Franco Verdi",
+        competenceUser: [
+            {
+                nameComp: 'js',
+                area: 'linguaggi programmazione',
+                level: 4
+            },
+            {
+                nameComp: 'Windows',
+                area: 'Sistemi operativi',
+                level: 1
+            }
+        ]
+    }
+];
+
+
+
+
+
+function nameORCompetence() {
+    console.log(document.getElementById("initialChoice").value)
```

Comment:
Non sarebbe comodo avere una funzione che prende in ingresso l'id e ti restituisce l'elemento? Migliorerebbe la leggibilità del codice e ridurrebbe la duplicazione del codice considerando che `documento.getElementById` lo usi in più parti del codice. Pensaci.